openapi/errata.js

```diff
@@ -97,7 +97,7 @@ const enumErrata = [
   { path: 'SessionAuthenticationMethod.kba', rename: 'knowledgeBased', renameReason: 'Legibility' },
   { path: 'SessionAuthenticationMethod.mfa', rename: 'multifactor', renameReason: 'Legibility' },
   { path: 'LogCredentialProvider.oktaAuthenticationProvider', rename: 'okta', renameReason: 'Legibility' },
-  { path: 'FactorType.webauthn', rename: 'webAuthentication', renameReason: 'Legibility' }
+  { path: 'FactorType.webauthn', rename: 'webAuthentication', renameReason: 'Legibility' },
```

Comment:
Does this not cause a JS error?

Reply:
hmm not sure what exactly you are seeing

Reply:
Oh sorry, the comma at the end